packages/terra-graphs/docs/helpers/Ticks.md

```diff
@@ -127,6 +127,8 @@ The tick values are set one of 3 methods, in the following priority:
 
 -   **Default**
 
-    If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
+    If neither `ticksCount` or custom tick values are prvided, then the default behavior is as follows:
+    
+    - If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.
 
-    If there is no Y2 axis, the Carbon will use D3.js to automatically find the tick values.
+    - If there is no Y2 axis and `ticksCount = undefined`, the Carbon will use D3.js to automatically find the tick values.
```

Comment:
is it possible to not fall under either of these conditions?

Reply:
I don't think there is a way for a consumer to get past all of these branches of logic.  Any graph with 1 or 2 Y-axes will qualify for at least one of these logic checks